pkg/infra/kubernetes/exec_unit.go

```diff
@@ -122,7 +125,7 @@ func (unit *HelmExecUnit) transformPod() (values []Value, err error) {
 	return
 }
 
-func (unit *HelmExecUnit) transformDeployment() ([]Value, error) {
+func (unit *HelmExecUnit) transformDeployment(cfg config.ExecutionUnit) ([]Value, error) {
```

Comment:
well need to make the same modification in transformPod because if they specify a pod manifest we wont add a deployment manifest